app/javascript/mastodon/features/account_timeline/components/familiar_followers.tsx

```diff
@@ -13,11 +13,13 @@ import { useAppDispatch, useAppSelector } from '@/mastodon/store';
 
 const AccountLink: React.FC<{ account?: Account }> = ({ account }) => {
   // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
Why disable the lint instead of using `??`?

Reply:
`account?.display_name` could be an empty string, so it needed the fallback to not always resolve to the left-hand side with nullish coalescing.

But I just realised that `display_name_html` is auto-populated with `.acct` if no display name is defined, so I can actually remove the entire line now and use it directly 🎉